telemetry/spans_test.go

```diff
@@ -147,3 +147,65 @@ func TestRecordSpanNilHarvester(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestEvents(t *testing.T) {
```

Comment:
Since Go doesn't have fancy automatic initializer functions for structs, this test is just checking whether Go can create fields that contain their assigned values. Let me know if I'm wrong!

Reply:
Yes, it's sort of tuning a guitar against itself. Being new to go, this test was mostly useful to check that my syntax was correct and that structs work the way I expect them to. 
I'd be happy to delete the test.